app.js

```diff
@@ -80,14 +81,16 @@ app.get('/wetty/ssh/:user', function(req, res) {
 });
 app.use('/', express.static(path.join(__dirname, 'public')));
 
+var port = process.env.NODE_ENV == 'production' ? 'systemd' : opts.port;
 if (runhttps) {
-    httpserv = https.createServer(opts.ssl, app).listen(opts.port, function() {
+    httpserv = https.createServer(opts.ssl, app).listen(
+      port, function() {
```

Comment:
nits: I think we should keep the coding style and code formatting of upstream, not phoxygen, when patching wetty. It will be much easier to follow upstream this way. We really want to have a minimal distance from them, I guess.